### DISCUSSION TITLE
Super AGI without "god mode"

### DISCUSSION THREAD

**Discussion by escondidocap** [TransformerOptimus/SuperAGI#434] [Q&A]
I would like to toggle between god mode and the basic mode as used in Auto GPT. Is there a way to do this where I can specify the number of runs and feed it more information/help answer questions it may have, etc? Some applications require more input through the process.

**Comment by DMTarmey** [comment]
You would need to look at all project like this autogpt, babyagi, GPT Engineer and more, find a platform that is already doing what you require as much of the work will already be done.
